bin/class-wcpay-changelog-formatter.php

```diff
@@ -0,0 +1,207 @@
+<?php
+/**
+ * Jetpack Changelogger Formatter for WooCommerce Payments
+ *
+ * @package WooCommerce\Payments
+ */
+
+use Automattic\Jetpack\Changelog\Changelog;
+use Automattic\Jetpack\Changelog\Parser;
+use Automattic\Jetpack\Changelogger\FormatterPlugin;
+use Automattic\Jetpack\Changelogger\PluginTrait;
+
+/**
+ * Jetpack Changelogger Formatter for WooCommerce Payments
+ *
+ * Class WCPay_Changelog_Formatter
+ */
+class WCPay_Changelog_Formatter extends Parser implements FormatterPlugin {
+	use PluginTrait;
+
+	/**
+	 * Bullet for changes.
+	 *
+	 * @var string
+	 */
+	private $bullet = '*';
+
+	/**
+	 * Output date format.
+	 *
+	 * @var string
+	 */
+	private $date_format = 'Y-m-d';
+
+	/**
+	 * String used as the date for an unreleased version.
+	 *
+	 * @var string
+	 */
+	private $unreleased = '2022-xx-xx';
```

Comment:
Can we get the year also via the current year so that it doesn't break?

Reply:
Yup, good idea. Addressed in 137f908